lib/AbstractHandler.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Phpactor\Extension;
+
+use Phpactor\LanguageServerProtocol\ClientCapabilities;
+
+abstract class AbstractHandler
+{
+    /**
+     * @var ClientCapabilities
+     */
+    protected $clientCapabilities;
+
+    public function __construct(ClientCapabilities $clientCapabilities)
+    {
+        $this->clientCapabilities = $clientCapabilities;
```

Comment:
I'd rather avoid Abstract classes. Not all handlers need to know the client capabilities, and chances are you need to override the constructor anyways.

Reply:
I can move back those duplicates if you want. 

Indeed, I forgotten to update some handlers to be depend on the client capabilities.